scripts/config

```diff
@@ -15,7 +15,7 @@ source _variables
 # RETRIEVE ARGUMENTS
 #=================================================
 
-app=${YNH_APP_INSTANCE_NAME:-$YNH_APP_ID}
+app=$YNH_APP_INSTANCE_NAME
```

Comment:
That line is actually, use $YNH_APP_INSTANCE_NAME, and if it's empty, use $YNH_APP_ID instead.
If you end up with $YNH_APP_ID, then you should check that PR to see if you actually got anything in $app.